ompi/mca/osc/rdma/osc_rdma_accumulate.c

```diff
@@ -179,8 +221,11 @@ static int ompi_osc_rdma_fetch_and_op_atomic (ompi_osc_rdma_sync_t *sync, const
 
     OSC_RDMA_VERBOSE(MCA_BASE_VERBOSE_TRACE, "initiating fetch-and-op using %d-bit btl atomics. origin: 0x%" PRIx64,
                      (4 == extent) ? 32 : 64, *((int64_t *) origin_addr));
-
-    origin = (8 == extent) ? ((int64_t *) origin_addr)[0] : ((int32_t *) origin_addr)[0];
+    if (8 == extent) {
```

Comment:
`extent` is always 4 or 8 so we can pass it to `osc_rdma_accelerator_mem_copy`.

Reply:
ACK